properties.schema

```diff
@@ -32,6 +33,7 @@
                     "_requireCourseCompleted": {
                       "type":"boolean",
                       "required":false,
+                      "default": "false",
```

Comment:
I think one of the completion criteria needs to be on by default, otherwise using the default settings will mean the course never gets set to completed. My suggestion would be for `_requireCourseCompleted` to be `true` by default